phoenicis-scripts/src/main/java/org/phoenicis/scripts/wizard/UiSetupWizardImplementation.java

```diff
@@ -47,12 +48,12 @@
 
     /**
      * Create the setupUi
-     *
-     * @param title title of the setupUi
+     *  @param title title of the setupUi
+     * @param miniature
      * @param userHome
      */
     public UiSetupWizardImplementation(String title,
```

Comment:
Additional JavaDoc for the remaining parameters, and maybe the class as a whole.